### DISCUSSION TITLE
Unable to "Unlock Network Settings" to use self-hosted server on system with OpenRC init system

### DISCUSSION THREAD

**Discussion by jahway603** [rustdesk/rustdesk#6442] [General]
### Bug Description

Unable to "Unlock Network Settings" to use self-hosted server on system with OpenRC init system

### How to Reproduce

1. Run rustdesk.
1. Navigate to Settings -> Network
1. Click on "Unlock network settings"
1. Nothing occurs.

### Expected Behavior

The expectation is that the user will be able to "Unlock network settings" and then configure their self-hosted rustdesk server.
I did not encounter this issue on a client system running the systemd init system.

### Operating system(s) on local side and remote side

Artix -> Debian

### RustDesk Version(s) on local side and remote side

1.2.3 client -> 1.1.8 server

### Screenshots

N/A

**Comment by jahway603** [comment]
Please add support for non-systemd Linux distros...

**Reply by jahway603** [reply]
Just tested this with client version 1.3.2 on Artix with OpenRC. It did prompt me for my password and the issue appears to now be resolved. Thank you.

**Comment by tienbuigia** [comment]
Yes, I have this issue as well.
I'm using Runit init system on Artix.

**Comment by nordic-style** [comment]
Same problem on macos 14.2.1 arm64 build.

**Reply by 21pages** [reply]
works on my 14.2.1

**Reply by nordic-style** [reply]
Does not work with 1.2.3 and 1.2.4 nightly (from 2024-01-23)

**Comment by ghost** [comment]


**Reply by nordic-style** [reply]
That did not help for me. I canot change any "unlock" settings since i cannot unlock them at all. Doesn't matter whether i use sudo or not.

**Comment by TwoClocks** [comment]
If you run from the cmd-line, you'll get a prompt to enter the root password when trying to unlock the network settings.

Seems like a bug. Seems like there must be some standard way to do this via a GUI by now? Also one that will use sudo if possible?

**Reply by d4munche3z** [reply]
I have the same issue tested with rustdesk x86_64 on Ubuntu 23. However, I was not able to get any command line prompt for password. 

Received this on terminal:
error checking for authorization com.rustdesk.rustdesk.options

**Reply by calim20** [reply]
ubuntu 24.10
rustdesk/now 1.3.0 amd64

Starting the client from the desktop I get asked for my password but nothing changes after I gave it.
Starting it from bash leads to the same.
Starting it with sudo from bash doesn't ask for password and nothing happens.
Also no usable output in the terminal when started from bash.

I guess I have to manually edit ~/.config/rustdesk/RustDesk2.toml

**Reply by 21pages** [reply]
Try https://github.com/rustdesk/rustdesk/releases/tag/1.3.2

**Reply by calim20** [reply]
Great! It works! :-) Thank you very much! I got so used to that everything gets updated by apt I totally overlooked the manual install. Isn't there any repo to use?
